### ISSUE TITLE
Loading state only starts when filters request are sent

### ISSUE THREAD

**Issue by xavijam** [CartoDB/deep-insights.js#234]
But they should start when first Map API request has been done.

cc @alonsogarciapablo @viddo

**Comment by xavijam** [comment]
And cc maybe @saleiva, @javisantana and @javierarce, becuase I feel there is a wait at the beginning in some examples, we could improve it. What do you think?

**Comment by alonsogarciapablo** [comment]
This is related to #226.

Why not show the loading bars as soon as possible (as soon as the widget is rendered)?

**Comment by xavijam** [comment]
For me it makes sense, but the problem is we should "trigger" the loading in the map instantiation, what do you think?

**Comment by saleiva** [comment]
+1 to what pablo says.

2016-02-22 14:55 GMT+01:00 Javier Álvarez Medina <notifications@github.com>:

> For me it makes sense, but the problem is we should "trigger" the loading
> in the map instantiation, what do you think?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/CartoDB/deep-insights.js/issues/234#issuecomment-187185424>
> .
>



-- 
Sergio Alvarez Leiva

**Comment by viddo** [comment]
Backbone have [both `request` (when a request is in process) and `sync` (when a response is returned) events](http://backbonejs.org/#Sync), triggered due to a fetch, which should be sufficient to render the appropriate state.

**Comment by xavijam** [comment]
No, the thing is we are not taking into account there is an important request before all the widgets logic, the maps API post, this is not a problem of backbone sync, request or whatever.
